api/types/v1/types.proto

```diff
@@ -71,3 +71,11 @@ enum TimeSeriesAggregationType {
   TIME_SERIES_AGGREGATION_TYPE_SUM = 0;
   TIME_SERIES_AGGREGATION_TYPE_AVERAGE = 1;
 }
+
+message FunctionSelector {
+  repeated Function stack_trace = 1;
```

Comment:
A question, would the user select a subtree of a stack trace or can they select functions that are not in the same call chain? Depending on the answer, `functions` could be a more flexible name for the field here.

Reply:
The stack trace points to the subtree root. In the future we will likely extend the selector to cover the case when all the function instances (all the stack traces containing the function) are to be selected. This is the reason why `FunctionSelector` is extendable: we can always add a new field (e.g `function`) that would serve for that purpose.

However, I don't really like names `FunctionSelector` and `Function` here, although they are quite precise (as of now). I'm thinking about renaming it to `LocationSelector` and `Location` respectively